BaseCollector.py

```diff
@@ -200,12 +200,12 @@ def generate_gauges(self, metric_type, calling_class, vrops_entity_name, labelna
             else:
                 return {}
         else:
-            return False
```

Comment:
Introduce some error message, that we are hitting this condition. Isn't it missing for the `stats` section too?

You can also skip the two `else` and just give it a `return {}` at the end with some message, that given metric type is not handled. If one of the `if`s are matching, it won't reach this point.

Reply:
Done.

Reply:
Done.